src/chrome/content/rules/yupoo.com.xml

```diff
@@ -13,7 +13,7 @@
 	<target host="pic.yupoo.com" />
 	<target host="photo.yupoo.com" />
 
-	<rule from="^http://photo\.yupoo\.com/" to="https://pic.yupoo.com/" />
+	<rule from="^http://photo\.yupoo\.com/" to="https://photo.yupoo.com/" />
```

Comment:
This is already covered [below](https://github.com/EFForg/https-everywhere/pull/14542/files#diff-50824c25fb193c3cb23dc91d5d83627aR19), you can simply remove this line.